recipes-core/packagegroups/packagegroup-prplmesh.bb

```diff
@@ -6,7 +6,9 @@
 DESCRIPTION = "Packages needed to support prplMesh"
 
 LICENSE = "BSD-2-Clause-Patent.txt"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
+# LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
+# LIC_FILES_CHKSUM = "file://BSD-2-Clause-Patent.txt;md5=0518d409dae93098cca8dfa932f3ab1b"
```

Comment:
Why string commented instead of removing?

Reply:
Solved in #14